drivers/gpio/gpio_efinix_sapphire_priv.h

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
If this header is not being used elsewhere in Zephyr, can it be merged into the driver?

Reply:
Alright, I have now combined it with the driver source.